src/components/filters/DateFilter/utils/DateConversions.ts

```diff
@@ -4,11 +4,32 @@ import isString = require("lodash/isString");
 import isDate = require("lodash/isDate");
 import { platformDateFormat } from "../../../../constants/Platform";
 
+const NUM_OF_MILISECONDS_IN_MINUTE = 60000;
 export const convertDateToPlatformDateString = (date: Date | undefined | null): string | undefined | null =>
     isDate(date) ? moment(date).format(platformDateFormat) : date;
 
 export const convertPlatformDateStringToDate = (
-    platformDate: string | undefined | null,
-): Date | undefined | null => (isString(platformDate) ? moment(platformDate).toDate() : platformDate);
+    platformDate: string | Date | undefined | null,
+): Date | undefined | null => {
+    if (!isString(platformDate)) {
+        return platformDate;
+    }
 
-export const createDateFromString = (dateString: string) => (dateString ? new Date(dateString) : undefined);
+    /**
+     * get Date time object with the same day in any local
+     * Example: "2019-11-28" local time is GTM-0600
+     *   local will display is "2019-11-27T18:00:00 GTM-0600"
+     *   after convert local will display is "2019-11-28T00:00:00 GTM-0600"
+     *
+     * Example: "2019-11-28" local time is GTM+0600
+     *   local will display is "2019-11-28T06:00:00 GTM+0600"
+     *   after convert local will display is "2019-11-28T00:00:00 GTM+0600"
+     */
```

Comment:
`get Date time object with the same day in any local`
=> what is `local`? Did you mean `localTimeOffset`?

`local will display is` 
=> `local will display`

`after convert`
=> `after converting`